packages/gatsby-cli/src/create-cli.js

```diff
@@ -10,6 +10,7 @@ const {
   setDefaultTags,
   setTelemetryEnabled,
 } = require(`gatsby-telemetry`)
+console.log(require.resolve(`gatsby-telemetry`))
```

Comment:
```suggestion
```

Reply:
So this ended up sneaking in after all, thanks for spotting it